lib/base/novacom.js

```diff
@@ -76,7 +76,7 @@ const async = require('async'),
          * @param {Function} next a common-JS callback invoked when the DB is ready to use.
          */
         load: function(next) {
-            log.verbose("Resolver#load()");
+            log.info("novacom#Resolver#load()");
```

Comment:
All "Resolver" change to "Resolver()"
and all "Session" change to "Session()"

I've been thinking a lot, and I need to change this file to be consistent with other files. We decided that changing this file would have the fewest correction points.

Reply:
Done.